README.md

```diff
@@ -1,3 +1,3 @@
-[!WARNING] 
+⚠️ WARNING: 
 We no longer support this SDK for Player integration on web environments. It can be used for interaction with the [Platform API](https://developers.dailymotion.com/api/platform-api/) only. 
 The Player should be embedded using the new default Player integration methods, info can be found [here](https://developer.dailymotion.com/sdk/player-sdk/web).
```

Comment:
typo here, it should be https://developers.dailymotion.com/sdk/player-sdk/web

and we good 🚀